src/Jaeger/Jaeger.csproj

```diff
@@ -7,7 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Jaeger.Thrift\Jaeger.Thrift.csproj" />
+      <ProjectReference Include="..\Jaeger.Grpc\Jaeger.Grpc.csproj" />
+      <ProjectReference Include="..\Jaeger.Thrift\Jaeger.Thrift.csproj" />
```

Comment:
Doesn't have to be part of this PR, but if one of your goal was "not to have thrift", doesn't it still get linked?

Reply:
Will be part of a separate PR specifically targetting #116 and done before I will release 0.4.0. So this state won't end up in any deployment.